src/components/SliderContainer.ts

```diff
@@ -1,213 +1,214 @@
+}
+
+interface SliderContainerProps extends WrapperProps {
+    bootstrapStyle: BootstrapStyle;
+    decimalPlaces: number;
+    maxAttribute: string;
+    minAttribute: string;
+    noOfMarkers: number;
+    onChangeMicroflow: string;
+    readOnly: boolean;
+    stepValue: number;
+    stepAttribute: string;
+    tooltipText: string;
+    valueAttribute: string;
+}
+
+interface SliderContainerState {
+    maximumValue?: number;
+    minimumValue?: number;
+    stepValue?: number;
+    value: number | null;
+}
+
+class SliderContainer extends Component<SliderContainerProps, SliderContainerState> {
+    private subscriptionHandles: number[];
+    private attributeCallback: (mxObject: mendix.lib.MxObject) => () => void;
+
+    constructor(props: SliderContainerProps) {
+        super(props);
+
+        this.state = this.updateValues(props.mxObject);
+        this.subscriptionHandles = [];
+        this.handleAction = this.handleAction.bind(this);
+        this.onUpdate = this.onUpdate.bind(this);
+        this.attributeCallback = mxObject => () => this.setState(this.updateValues(mxObject));
+    }
+
+    render() {
+        const disabled = !this.props.mxObject
+            || !!(this.props.stepAttribute && this.props.mxObject.isReadonlyAttr(this.props.stepAttribute));
+
+        const alertMessage = !disabled ? SliderContainer.validateSettings(this.state) || this.validateValues() : "";
+
+        return createElement(Slider, {
+            alertMessage,
+            bootstrapStyle: this.props.bootstrapStyle,
+            className: this.props.class,
+            decimalPlaces: this.props.decimalPlaces,
+            disabled,
+            maxValue: this.state.maximumValue,
+            minValue: this.state.minimumValue,
+            noOfMarkers: this.props.noOfMarkers,
+            onChange: this.handleAction,
+            onUpdate: this.onUpdate,
+            stepValue: this.state.stepValue,
+            style: SliderContainer.parseStyle(this.props.style),
+            tooltipText: this.props.tooltipText,
+            value: this.state.value
+        });
+    }
+
+    componentWillReceiveProps(newProps: SliderContainerProps) {
+        this.resetSubscriptions(newProps.mxObject);
+        this.setState(this.updateValues(newProps.mxObject));
+    }
+
+    componentWillUnmount() {
+        this.subscriptionHandles.forEach(window.mx.data.unsubscribe);
+    }
+
+    public static parseStyle(style = ""): {[key: string]: string} {
+        try {
+            return style.split(";").reduce<{[key: string]: string}>((styleObject, line) => {
+                const pair = line.split(":");
+                if (pair.length === 2) {
+                    const name = pair[0].trim().replace(/(-.)/g, match => match[1].toUpperCase());
+                    styleObject[name] = pair[1].trim();
+                }
+                return styleObject;
+            }, {});
+        } catch (error) {
+            console.log("Failed to parse style", style, error);

```

Comment:
Check your linter is up to date and add
`// tslint:disable-next-line no-console`